doc/userdoc/conf.py

```diff
@@ -39,10 +39,13 @@
 else:
     source_dir = Path(__file__).resolve().parent.parent.parent.resolve()
 
-doc_build_dir = Path(os.environ['OLDPWD']) / 'doc/userdoc'
 
-if os.environ.get('READTHEDOCS', 'False') == 'True':
+read_the_docs_build = os.environ.get('READTHEDOCS') == 'True'
+
+if read_the_docs_build:
     doc_build_dir = source_dir / 'doc/userdoc'
+else:
+    doc_build_dir = Path(os.environ['OLDPWD']) / 'doc/userdoc'
```

Comment:
I'd avoid the local variable and use double quotes in accordance to our style guide ;):

```suggestion
if os.environ.get("READTHEDOCS") == "True":
    doc_build_dir = source_dir / "doc/userdoc"
else:
    doc_build_dir = Path(os.environ["OLDPWD"]) / "doc/userdoc"
```